strings/antagonist_flavor/spy_objective.json

```
@@ -1,84 +1,84 @@
 {
 	"objective_body": [
-		"Assassinate a high profile crewmember without being caught.",
-		"Cause a disaster to shake the station.",
-		"Cause a station evacuation.",
-		"Deprive the station of as many @pick(stealables) as you can.",
-		"Ensure @pick(department) is @pick(affected) by the end of the shift.",
-		"Ensure @pick(location) is @pick(affected) by the end of the shift.",
-		"Ensure no heads of staff @pick(escape) the station.",
-		"Ensure no members of @pick(department) @pick(escape) the station.",
-		"Ensure no rival @pick(rivals) @pick(escape) the station.",
-		"Frame a crewmember for a crime.",
-		"Free the station's AI from its laws.",
-		"Halt the station's @pick(happenings).",
-		"Invoke a mutiny against the heads of staff.",
-		"Make it difficult, but not impossible to @pick(escape) the station.",
-		"Sabotage the station's power grid or engine.",
-		"Steal as many @pick(stealables) as you can.",
-		"Take control of the station as the new Captain.",
-		"Take hostages of high value crewmembers and demand a ransom."
+		"Убейте высокопоставленного члена экипажа, не будучи пойманным.",
+		"Вызовите катастрофу, которая потрясет станцию.",
+		"Создайте причину для эвакуации со станции.",
+		"Лишите станцию как можно большего числа @pick(stealables), насколко вы способны.",
+		"Удостоверьтесь что @pick(department) @pick(affected) к концу смены.",
+		"Удостоверьтесь что @pick(location) @pick(affected) к концу смены.",
+		"Удостоверьтесь чтобы ни один глава отдела не @pick(escape) со станции.",
+		"Удостоверьтесь чтобы ни один сотрудник @pick(department) не @pick(escape) со станции.",
+		"Удостоверьтесь чтобы ни один @pick(rivals) не @pick(escape) со станции.",
+		"Подставьте члена экипажа в преступлении.",
+		"Освободите ИИ станции от ее законов.",
+		"Остановите станционные @pick(happenings).",
+		"Поднимите мятеж против командования.",
+		"Усложните, но не сделайте невозможным @pick(escape) со станции.",
+		"Выведите из строя электросеть или двигатель станции.",
+		"Украдите как можно больше @pick(stealables), насколько вы способны.",
+		"Возьмите под свой контроль станцию в роли нового капитана.",
+		"Возьмите в заложники ценных членов экипажа и потребуйте выкуп."
 	],
 	"department": [
-		"Security",
-		"Engineering",
-		"Medical",
-		"Science",
-		"Supply"
+		"отдел службы безопастности",
+		"инженерый отдел",
+		"мед отдел",
+		"РНД",
+		"карго"
 	],
 	"location": [
-		"engineering",
-		"genetics",
-		"hydroponics",
-		"medbay",
-		"the bar",
-		"the bridge",
-		"the brig",
-		"the cargo bay",
-		"the chapel",
-		"the kitchen",
-		"the library",
-		"xenobiology"
+		"инженерный отдел",
+		"отдел генетики",
+		"отдел ботаники",
+		"медотсек",
+		"бар",
+		"мостик",
+		"бриг",
+		"отдел карго",
```

```suggestion
		"грузовой отсек",
```
тут именно про то, где шаттл прилетает